### PR TITLE
Feature persistmsg

### PR DESCRIPTION
**Pull request by urviljoshi**

### CODE REVIEW

File: Ballerina.lock [hunk 1]
```diff
@@ -8,16 +8,16 @@ org_name = "mosip"
 name = "services"
```
Comment (vishwa-vyom) [inline Ballerina.lock:8]: Is Ballerina.lock generated when the code is built?

File: ballerina-internal.log [hunk 2]
```diff
@@ -488,3 +488,851 @@ java.lang.NullPointerException
 	at java.util.Optional.ifPresent(Optional.java:159)
 	at org.ballerinalang.tool.Main.main(Main.java:57)
  
+[2021-01-27 19:36:56,929] SEVERE {b7a.log.crash} - null
```
Comment (vishwa-vyom) [inline ballerina-internal.log:491]: Do we need to commit this ballerina-internal.log file?

File: src/hubinit/failedcontentpull.bal [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import mosip/repository;
+import ballerina/lang.'int;
+import ballerina/config;
+
+@http:ServiceConfig {
+    basePath: "/websub/hub"
+}
+service failedcontentpull on hubListener {
+
+    resource function failedmessage(http:Caller caller,
```
Comment (vishwa-vyom) [inline src/hubinit/failedcontentpull.bal:11]: Can this be camel case or capital camel case as per Ballerina coding standards?

Comment (vishwa-vyom) [inline src/hubinit/failedcontentpull.bal:11]: Can this be camel case as per Ballerina coding standards?

File: ballerina-internal.log (hunk 2 above)
Comment (vishwa-vyom) [reply ballerina-internal.log:491]: Remove this and add to git ignore

File: src/hubinit/failedcontentpull.bal (hunk 3 above)
Comment (urviljoshi) [reply src/hubinit/failedcontentpull.bal:11]: yes

Comment (urviljoshi) [reply src/hubinit/failedcontentpull.bal:11]: yes

**Pull request closed without merging**